data/autoyast_sle15/autoyast_nfs_share.xml

```diff
@@ -0,0 +1,231 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <suse_register>
+    <do_registration config:type="boolean">true</do_registration>
+    <email/>
+    <reg_code>{{SCC_REGCODE}}</reg_code>
+    <install_updates config:type="boolean">true</install_updates>
+    <reg_server>{{SCC_URL}}</reg_server>
+    <addons config:type="list">
+      <addon>
+        <name>sle-module-basesystem</name>
+        <version>{{VERSION}}</version>
+        <arch>{{ARCH}}</arch>
+      </addon>
+      <addon>
+        <name>sle-module-server-applications</name>
+        <version>{{VERSION}}</version>
+        <arch>{{ARCH}}</arch>
+      </addon>
+      <addon>
+        <name>sle-module-desktop-applications</name>
```

Comment:
Do we really need this module to be added? I guess we could save some installation time if use only 2 other modules, as we test nfs share here.

Reply:
Good point! I didn't make it when doing some manual testing but it worked after I removed the following nodes:
```
<name>sle-module-desktop-applications</name>
<default_target>graphical</default_target>
<package>sle-module-desktop-applications-release</package>
<pattern>gnome_basic</pattern>
<pattern>gnome_basis</pattern>
<pattern>minimal_base</pattern>
<pattern>x11</pattern>
<pattern>x11_enhanced</pattern> 
```
Updated VR.